### PR TITLE
Add minikube/onprem install quickstart to the quickstart repo

### PR DESCRIPTION
**Pull request by 0-sv**
Closes https://github.com/redkubes/unassigned-issues/issues/359.

Critique welcome.

### CODE REVIEW

File: onprem/README.md [hunk 1]
```diff
@@ -0,0 +1,79 @@
+# Table of Contents
+
+- [Table of Contents](#table-of-contents)
```
Comment (diabhey) [inline onprem/README.md:3]: @0-sv TOC within TOC can be removed

**Comment by diabhey** [comment]
@0-sv: Let me test it locally and get back to you.

**Comment by 0-sv** [comment]
Ok, but use a beefy server if you can, it's not recommended on your laptop. Good luck and I am curiously awaiting your findings :).

File: onprem/README.md (hunk 1 above)
Comment (0-sv) [reply onprem/README.md:3]: If we want to use the ToC auto-generation feature, it keeps popping up if you use a markdown header. Let me know if this is OK, if you want to remove the ToC or do it manually.

**Review by diabhey — APPROVED** [review]
Merging the changes as it looks good to me. I will later test it on a beefy server.

**Pull request merged**